src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -56,7 +58,7 @@ const SearchBarDropdownSection = ({
       </Row>
       <Column gap="12">
         {suggestions.map((suggestion, index) =>
-          isLoading ? (
+          isLoading || !suggestion ? (
```

Comment:
Yes, suggestion are actually undefined while loading and can potentially be undefined from graphql. This would cause the app to crash when I migrated the data source as several components didn't expect it to be undefined.